wire-ios/Wire-iOS/Sources/AppDelegate.swift

```diff
@@ -71,7 +71,8 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
     private(set) var launchType: ApplicationLaunchType = .unknown
 
     // MARK: - Public Set Property
-    var window: UIWindow?
+
+    private(set) var mainWindow: UIWindow!
```

Comment:
I called the property `mainWindow` because `window` collides with the declaration in `UIApplicationDelegate` and hence it wouldn't be possible to make it `private(set)`.
Also `keyWindow` wouldn't be correct, since we use other temporary windows (e.g. screen curtain) and make them key and visible.